Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -0,0 +1,1283 @@
+                    }
+
+                    if (individualInterfaces)
+                    {
+                        return new NetCollection(_interfaceAddresses);
+                    }
+
+                    // No bind address and no exclusions, so listen on all interfaces.
+                    NetCollection result = new NetCollection();
+
+                    if (IsIP4Enabled)
+                    {
+                        result.Add(IPAddress.Any);
+                    }
+
+                    if (IsIP6Enabled)
+                    {
+                        result.Add(IPAddress.IPv6Any);
+                    }
+
+                    return result;
+                }
+
+                // Remove any excluded bind interfaces.
+                return _bindAddresses.Exclude(_bindExclusions);
+            }
+        }
+
+        /// <inheritdoc/>
+        public string GetBindInterface(string source, out int? port)
+        {
+            if (!string.IsNullOrEmpty(source))
+            {
+                if (IPHost.TryParse(source, out IPHost host))
+                {
+                    return GetBindInterface(host, out port);
+                }
+            }
+
+            return GetBindInterface(IPHost.None, out port);
+        }
+
+        /// <inheritdoc/>
+        public string GetBindInterface(IPAddress source, out int? port)
+        {
+            return GetBindInterface(new IPNetAddress(source), out port);
+        }
+
+        /// <inheritdoc/>
+        public string GetBindInterface(HttpRequest source, out int? port)
+        {
+            string result;
+
+            if (source != null && IPHost.TryParse(source.Host.Host, out IPHost host))
+            {
+                result = GetBindInterface(host, out port);
+                port ??= source.Host.Port;
+            }
+            else
+            {
+                result = GetBindInterface(IPNetAddress.None, out port);
+                port ??= source?.Host.Port;
+            }
+
+            return result;
+        }
+
+        /// <inheritdoc/>
+        public string GetBindInterface(IPObject source, out int? port)
+        {
+            port = null;
+            bool isChromeCast = source == IPNetAddress.IP4Loopback;
+            // Do we have a source?
+            bool haveSource = !source.Address.Equals(IPAddress.None);
+            bool isExternal = false;
+
+            if (haveSource)
+            {
+                if (!IsIP6Enabled && source.AddressFamily == AddressFamily.InterNetworkV6)
+                {
+                    _logger.LogWarning("IPv6 is disabled in JellyFin, but enabled in the OS. This may affect how the interface is selected.");
+                }
+
+                if (!IsIP4Enabled && source.AddressFamily == AddressFamily.InterNetwork)
+                {
+                    _logger.LogWarning("IPv4 is disabled in JellyFin, but enabled in the OS. This may affect how the interface is selected.");
```

Comment:
```suggestion
                    _logger.LogWarning("IPv4 is disabled in Jellyfin, but enabled in the OS. This may affect how the interface is selected.");
```